homeassistant/components/shelly/event.py

```diff
@@ -71,6 +73,10 @@ class ShellyRpcEventDescription(EventEntityDescription):
         config, status, key
     ),
 )
+RPC_TEST_EVENT: Final = ShellyRpcEventDescription(
+    key="smoke",
+    translation_key="smoke",
```

Comment:
What's it for if it's not used?

Reply:
I basically copied this from the other definitions, but I could not find out how it was used.

Reply:
The translation key is needed when you want to add ability to translate the entity name or use a custom icon. I think we don't need this here.